<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmWaveBar</class>
 <widget class="QWidget" name="frmWaveBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="WaveBar" name="waveBar1"/>
   </item>
   <item>
    <widget class="WaveBar" name="waveBar2"/>
   </item>
   <item>
    <widget class="WaveBar" name="waveBar3"/>
   </item>
   <item>
    <widget class="WaveBar" name="waveBar4"/>
   </item>
   <item>
    <widget class="WaveBar" name="waveBar5"/>
   </item>
   <item>
    <widget class="WaveBar" name="waveBar6"/>
   </item>
   <item>
    <widget class="WaveBar" name="waveBar7"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WaveBar</class>
   <extends>QWidget</extends>
   <header>wavebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
